### PR TITLE
added: as a user i can view a spaceships page

### PR DESCRIPTION
**Pull request by Dabbous1**
added the show.html.erb file for spaceship and added a show method in spaceship controller - pending review

### CODE REVIEW

**Comment by Dabbous1** [comment]
[As a user, I can view a spaceship's page](https://trello.com/c/tHWwKN4w/3-as-a-user-i-can-view-a-spaceships-page)

File: db/seeds.rb [hunk 1]
```diff
@@ -5,3 +5,86 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+user = User.create!({email: "aly@gmail.com", password: "123478878", first_name: "Omar", last_name: "Youssef"})
```
Comment (pimmesz) [inline db/seeds.rb]: Do we want to include this seed for everyone? Or is it just for testing?

Comment (Stefanode) [reply db/seeds.rb]: testing

File: app/controllers/spaceships_controller.rb [hunk 2]
```diff
@@ -3,6 +3,7 @@ def index
   end
 
   def show
+    @spaceship = Spaceship.find(params[:id])
```
[the remark this answers is not in the document]
Comment (Stefanode) [reply app/controllers/spaceships_controller.rb]: ok thx

File: config/webpack/environment.js [hunk 3]
```diff
@@ -1,45 +1,3 @@
 const { environment } = require('@rails/webpacker')
 
-// Bootstrap 3 has a dependency over jQuery:
```
[the remark this answers is not in the document]
Comment (Stefanode) [reply config/webpack/environment.js]: aut

File: db/seeds.rb [hunk 4]
```diff
@@ -5,3 +5,86 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
```
[the remark this answers is not in the document]
Comment (Stefanode) [reply db/seeds.rb]: We use faker

File: config/webpack/environment.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (Dabbous1) [reply config/webpack/environment.js]: this got removed automatically because the boostrap no longer has a dependency over jQuery in webpacker

**Pull request merged**